Doc/glossary.rst

```diff
@@ -1017,17 +1030,16 @@ Glossary
       :func:`bytes.splitlines` for an additional use.
 
    variable annotation
-      A type metadata value associated with a module global variable or
-      a class attribute. Its syntax is explained in section :ref:`annassign`.
-      Annotations are stored in the :attr:`__annotations__` special
-      attribute of a class or module object and can be accessed using
-      :func:`typing.get_type_hints`.
+      An :term:`annotation` of a global variable, or class variable.
+
+      For example, this variable is annotated as an int::
```

Comment:
I would reformulate it as `this variable is annotated as taking :class:int values` or similar.

Reply:
✔️ Done